ReleaseProcessAutomation/Steps/PipelineSteps/ReleaseAlphaBetaStep.cs

```diff
@@ -94,7 +95,34 @@ public void Execute (SemanticVersion nextVersion, string? commitHash, bool pause
 
     var nextJiraVersion = InputReader.ReadVersionChoice("Please choose next version (open JIRA issues get moved there):", nextPossibleJiraVersions);
 
-    _ijIraFunctionality.CreateAndReleaseJiraVersion(nextVersion, nextJiraVersion);
+    _jiraFunctionality.CreateAndReleaseJiraVersion(nextVersion, nextJiraVersion);
+
+    var currentFullVersion = nextVersion.GetCurrentFullVersion();
+    try
+    {
+      var jiraClosedIssuesWithFixVersion = _jiraFunctionality.GetClosedVersionsSummaryWithExactFixVersion(currentFullVersion);
+      
+      if (jiraClosedIssuesWithFixVersion.Any())
+      {
+        foreach (var issue in jiraClosedIssuesWithFixVersion)
+        {
+          Console.WriteLine($"{issue.Key} - {issue.Fields.Summary}");
+        }
+        Console.WriteLine(
+            $"Do you want to move the closed versions with Jira fixVersion '{currentFullVersion}' to the newly released version '{nextVersion}'?");
+        if (InputReader.ReadConfirmation())
+          if (_jiraFunctionality.MoveClosedIssuesToNewVersion(currentFullVersion, nextVersion))
+          {
+            Console.WriteLine($"Could not move closed jira issues from version '{currentFullVersion}' to '{nextVersion}'. \nDo you wish to continue?");
+            if (!InputReader.ReadConfirmation())
+              return;
+          }
```

Comment:
Style: If a statement spans across multiple lines braces should be added:

```suggestion
        if (InputReader.ReadConfirmation())
        {
          if (_jiraFunctionality.MoveClosedIssuesToNewVersion(currentFullVersion, nextVersion))
          {
            Console.WriteLine($"Could not move closed jira issues from version '{currentFullVersion}' to '{nextVersion}'. \nDo you wish to continue?");
            if (!InputReader.ReadConfirmation())
              return;
          }
        }
```